src/styles/blocks/main.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider using an SCSS variable for the font family. This will make it easier to update the font across the entire project if needed.